README.md

```diff
@@ -16,3 +17,26 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+## My Analysis
+
+To test the claim, I would use many different possible arrays such as: empty, sorted, reverse sorted, random, and partially sorted
+
+I would use these different arrays and use different lengths of each of them. I would use timing code to run the sorting algorithm with the different arrays then probably put the results into an excel spreadsheet and find the ratio of elements:time that every datapoint should be near. I would probably also graph the datapoints and look at the line of best fit and where the points fall just for another way to analyze it.
```

Comment:
...and what would you expect to see with the ratios and graphs?

Reply:
I have gone into more depth about what my test would show